MSI/Day11/비밀지도.js

```diff
@@ -0,0 +1,8 @@
+function solution(n, arr1, arr2) {
+    const answer = arr1.map((num1, i) => {
+        const combined = (num1 | arr2[i]).toString(2).padStart(n, '0');
+        return combined.replace(/0/g, ' ').replace(/1/g, '#');
+    });
```

Comment:
```suggestion
    const answer = arr1.map((num, i) => {
        const combined = (num | arr2[i]).toString(2).padStart(n, '0');
        return combined.replace(/0/g, ' ').replace(/1/g, '#');
    });
```
`arr1`에서 반복문을 돌린거라 `num1`인가요? 😂 개인적으로는 그냥 숫자를 뜻하는 `num`같은 네이밍으로 해도 괜찮을 것 같네요

Reply:
num 으로 수정했습니다 감사합니다!